Mathlib/CategoryTheory/FintypeCat.lean

```diff
@@ -76,11 +76,9 @@ theorem id_apply (X : FintypeCat) (x : X) : (𝟙 X : X → X) x = x :=
 theorem comp_apply {X Y Z : FintypeCat} (f : X ⟶ Y) (g : Y ⟶ Z) (x : X) : (f ≫ g) x = g (f x) :=
   rfl
 
-@[simp]
 lemma hom_inv_id_apply {X Y : FintypeCat} (f : X ≅ Y) (x : X) : f.inv (f.hom x) = x :=
   congr_fun f.hom_inv_id x
```

Comment:
perhaps best to replace the proof with `by simp`, so it is clear why this isn't a simp lemma

Reply:
We would actually need to import `CategoryTheory.Elementwise` into this file to make the proof `by simp`. I'll add a comment instead.